### PR TITLE
Create automated tests for running 35K scatter test in Alpha Environment [BA-6623]

### PR DESCRIPTION
**Pull request by grsterin**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by kshakir — APPROVED** [review]
LGTM

I cloned the repo and looked for similar files. The best I can come up with is `impossible-to-read-carbonited-metadata-directly-from-within-workflow-check.sh` so I assume this is heavily borrowed from that, and this looks like similar code to me.

**Comment by grsterin** [comment]
> LGTM
> 
> I cloned the repo and looked for similar files. The best I can come up with is `impossible-to-read-carbonited-metadata-directly-from-within-workflow-check.sh` so I assume this is heavily borrowed from that, and this looks like similar code to me.

True. The "impossible-to-read..." test was also created by me a while ago.

File: automation/successfully-run-35k-scatter-workflow-check.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source ./submission-perf-inc.sh
+
+ENV=alpha
+
+# this `true` which goes as a third parameter from the end on the following line is to enable call-caching
+launchSubmission dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace 35k-testing wide_scatter "" "" true false ""
+findLastSubmissionID dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace
+submission1=$submissionID
+
+waitForSubmissionAndWorkflowStatus 20 Done Succeeded dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace "$submission1"
```
Comment (mcovarr) [inline automation/successfully-run-35k-scatter-workflow-check.sh]: TOL is 20 attempts going to be sufficient for a workflow with this many jobs even if they all cache?

**Review by mcovarr — APPROVED** [review]

File: automation/successfully-run-35k-scatter-workflow-check.sh (hunk 1 above)
Comment (grsterin) [reply automation/successfully-run-35k-scatter-workflow-check.sh]: Indeed, looks like it takes hours even with call caching. Currently running it for the second time to find out how long will it take to complete.

Comment (grsterin) [reply automation/successfully-run-35k-scatter-workflow-check.sh]: Increased waiting time to 3 hours (180 attempts)

**Pull request merged**